.gitignore

```diff
@@ -27,5 +27,3 @@ coverage/
 # npm debug log:
 npm-debug.log
 
-# kubeconfig for cluster made by kind:
-kind.kubeconfig*
```

Comment:
Why was it removed?

Reply:
Because moved its destination directory to `/tmp` on container.

Reply:
Will not it be a problem when developing locally without a container?

Reply:
kubeconfig for `kind` was originally created into `${HOME}/.kube/config` by `aio/scripts/start-cluster.sh` https://github.com/kubernetes/dashboard/blob/master/aio/scripts/conf.sh#L81
`./kind.kubeconfig` was created for only development container.

Reply:
Okay, thank you.